theme_loftspace/__manifest__.py

```diff
@@ -55,20 +60,28 @@
         'static/description/loftspace_screenshot.jpg',
     ],
     'images_preview_theme': {
-        'website.s_cover_default_image': '/theme_loftspace/static/src/img/snippets/s_cover.jpg',
-        'website.s_banner_default_image': '/theme_loftspace/static/src/img/snippets/s_banner.jpg',
-        'website.s_three_columns_default_image_1': '/theme_loftspace/static/src/img/snippets/library_image_11.jpg',
-        'website.s_three_columns_default_image_2': '/theme_loftspace/static/src/img/snippets/library_image_13.jpg',
+        'website.s_image_text_default_image': '/theme_loftspace/static/src/img/snippets/s_image_text.jpg',
+        'website.s_product_list_default_image_1': '/theme_loftspace/static/src/img/snippets/s_product_1.jpg',
+
+        'website.s_picture_default_image': '/theme_loftspace/static/src/img/snippets/s_picture.jpg',
+
+        'website.s_media_list_default_image_1': '/theme_loftspace/static/src/img/snippets/s_media_list_1.jpg',
+        'website.s_media_list_default_image_2': '/theme_loftspace/static/src/img/snippets/s_media_list_2.jpg',
+        'website.s_text_image_default_image': '/theme_loftspace/static/src/img/snippets/s_text_image.jpg',
+
+        'website.s_cta_box_default_image': '/theme_loftspace/static/src/img/content/s_cta_box_default_image.jpg',
+
         'website.s_three_columns_default_image_3': '/theme_loftspace/static/src/img/snippets/library_image_07.jpg',
         'website.library_image_03': '/theme_loftspace/static/src/img/snippets/s_images_wall_01.jpg',
         'website.library_image_02': '/theme_loftspace/static/src/img/snippets/s_images_wall_05.jpg',
         'website.library_image_10': '/theme_loftspace/static/src/img/snippets/s_images_wall_06.jpg',
         'website.library_image_05': '/theme_loftspace/static/src/img/snippets/s_images_wall_02.jpg',
+        'website.library_image_07': '/theme_loftspace/static/src/img/snippets/library_image_07.jpg',
         'website.library_image_16': '/theme_loftspace/static/src/img/snippets/s_images_wall_04.jpg',
         'website.library_image_13': '/theme_loftspace/static/src/img/snippets/s_images_wall_03.jpg',
     },
     'configurator_snippets': {
-        'homepage': ['s_cover', 's_three_columns', 's_title', 's_images_wall', 's_call_to_action'],
+        'homepage': ['s_banner', 's_text_block', 's_striped', 's_key_images', 's_features', 's_quotes_carousel', 's_faq_collapse', 's_cta_box',],
```

Comment:
`s_call_to_action` was removed, but a template that extends `configurator_s_call_to_action` is still defined. Because of this, an error is raised when creating a new website through the configurator using this theme (e.g. "furniture repair shop" + center suggestion)